ui/pages/confirmations/hooks/useTrackERC20WithoutDecimalInformation.ts

```diff
@@ -51,7 +51,7 @@ const useTrackERC20WithoutDecimalInformation = (
         });
       }
     }
-  }, [tokenDetails, chainId, tokenAddress, trackEvent]);
```

Comment:
This hook getting `tokenDetails` from the result of `useGetTokenStandardAndDetails` hook.
And `useGetTokenStandardAndDetails` is using async result in itself. 

Doesn't that mean we want to have `tokenDetails` in the dependency array here?